<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>781</width>
      <height>501</height>
     </rect>
    </property>
    <property name="plainText">
     <string>Hello Wold!</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Save"/>
    <addaction name="action_SaveAnother"/>
    <addaction name="action_Open"/>
    <addaction name="separator"/>
    <addaction name="action_Close"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="action_Copy"/>
    <addaction name="action_Paste"/>
    <addaction name="action_Cut"/>
    <addaction name="separator"/>
    <addaction name="action_SlelctAll"/>
    <addaction name="action_Clear"/>
    <addaction name="separator"/>
    <addaction name="action_Undo"/>
    <addaction name="action_Redo"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>格式</string>
    </property>
    <widget class="QMenu" name="menuLanguage">
     <property name="title">
      <string>Language</string>
     </property>
     <addaction name="action_English"/>
     <addaction name="action_Chinese"/>
    </widget>
    <addaction name="action_Bold"/>
    <addaction name="action_Italic"/>
    <addaction name="action_UnderLine"/>
    <addaction name="separator"/>
    <addaction name="menuLanguage"/>
    <addaction name="separator"/>
    <addaction name="action_Text"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Save"/>
   <addaction name="action_New"/>
   <addaction name="action_SaveAnother"/>
   <addaction name="separator"/>
   <addaction name="action_Cut"/>
   <addaction name="action_Copy"/>
   <addaction name="action_Paste"/>
   <addaction name="action_SlelctAll"/>
   <addaction name="action_Clear"/>
   <addaction name="separator"/>
   <addaction name="action_Undo"/>
   <addaction name="action_Redo"/>
   <addaction name="separator"/>
   <addaction name="action_Bold"/>
   <addaction name="action_Italic"/>
   <addaction name="action_UnderLine"/>
   <addaction name="action_English"/>
   <addaction name="action_Chinese"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Open">
   <property name="icon">
    <iconset>
     <normalon>images/001.GIF</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="toolTip">
    <string>打开文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset>
     <normaloff>images/104.bmp</normaloff>images/104.bmp</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>保存当前文件</string>
   </property>
  </action>
  <action name="action_New">
   <property name="icon">
    <iconset>
     <normaloff>images/100.bmp</normaloff>images/100.bmp</iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
   <property name="toolTip">
    <string>新建文件</string>
   </property>
  </action>
  <action name="action_SaveAnother">
   <property name="icon">
    <iconset>
     <normaloff>images/106.bmp</normaloff>images/106.bmp</iconset>
   </property>
   <property name="text">
    <string>另存为</string>
   </property>
   <property name="toolTip">
    <string>另存为</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="icon">
    <iconset>
     <normaloff>images/132.bmp</normaloff>images/132.bmp</iconset>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
   <property name="toolTip">
    <string>关闭程序</string>
   </property>
  </action>
  <action name="action_Copy">
   <property name="icon">
    <iconset>
     <normaloff>images/202.bmp</normaloff>images/202.bmp</iconset>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
   <property name="toolTip">
    <string>复制文本</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_Paste">
   <property name="icon">
    <iconset>
     <normaloff>images/204.bmp</normaloff>images/204.bmp</iconset>
   </property>
   <property name="text">
    <string>粘贴</string>
   </property>
   <property name="toolTip">
    <string>粘贴文本</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_SlelctAll">
   <property name="text">
    <string>全选</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="action_Clear">
   <property name="text">
    <string>清除</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="action_Undo">
   <property name="icon">
    <iconset>
     <normaloff>images/206.bmp</normaloff>images/206.bmp</iconset>
   </property>
   <property name="text">
    <string>撤销</string>
   </property>
   <property name="toolTip">
    <string>撤销上次操作</string>
   </property>
  </action>
  <action name="action_Redo">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/208.bmp</normaloff>images/208.bmp</iconset>
   </property>
   <property name="text">
    <string>重做</string>
   </property>
   <property name="toolTip">
    <string>重做上次操作</string>
   </property>
  </action>
  <action name="action_Bold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/500.bmp</normaloff>images/500.bmp</iconset>
   </property>
   <property name="text">
    <string>粗体</string>
   </property>
   <property name="toolTip">
    <string>字体加粗</string>
   </property>
  </action>
  <action name="action_Italic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/502.bmp</normaloff>images/502.bmp</iconset>
   </property>
   <property name="text">
    <string>斜体</string>
   </property>
   <property name="toolTip">
    <string>字体倾斜</string>
   </property>
  </action>
  <action name="action_UnderLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/504.bmp</normaloff>images/504.bmp</iconset>
   </property>
   <property name="text">
    <string>下划线</string>
   </property>
  </action>
  <action name="action_English">
   <property name="icon">
    <iconset>
     <normaloff>images/timg2.jpg</normaloff>images/timg2.jpg</iconset>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="action_Chinese">
   <property name="icon">
    <iconset>
     <normaloff>images/CN.jpg</normaloff>images/CN.jpg</iconset>
   </property>
   <property name="text">
    <string>中文</string>
   </property>
  </action>
  <action name="action_Text">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>显示按钮文字</string>
   </property>
  </action>
  <action name="action_Cut">
   <property name="icon">
    <iconset>
     <normaloff>images/200.bmp</normaloff>images/200.bmp</iconset>
   </property>
   <property name="text">
    <string>剪切</string>
   </property>
   <property name="toolTip">
    <string>剪切文本</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>action_Copy</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>plainTextEdit</sender>
   <signal>undoAvailable(bool)</signal>
   <receiver>action_Undo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>400</x>
     <y>317</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>plainTextEdit</sender>
   <signal>redoAvailable(bool)</signal>
   <receiver>action_Redo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>400</x>
     <y>317</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Cut</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>cut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Paste</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_SlelctAll</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Clear</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Close</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Redo</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>redo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Undo</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>undo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
